README.adoc

```diff
@@ -63,8 +63,8 @@ include::complete/src/main/java/com/example/testingrestdocs/HomeController.java[
 
 == Run the Application
 
-The Spring Initializr creates a `main` class that you can use to launch the application.
-The following listing (from
+The Spring Initializr creates a `{YourProjectName}Application` class with a `main` function 
```

Comment:
Technically it's a "method", not a "function" in Java

Reply:
Good catch. I've fixed it in commit e12240d.